lib/utils.js

```diff
@@ -430,6 +431,20 @@ function deepMerge(base, extras) {
 }
 
 /**
+ * Properties deep merge
+ * Similar as deepMerge but also works on single properties of any type
+ *
+ * @param {Object} base The base property
+ * @param {Object} extras The property to merge with base
+ * @returns {Object} The merged property
+ */
+function deepMergeProperty(base, extras) {
+  let mergedObject = deepMerge({key: base}, {key: extras});
+  let mergedProperty = mergedObject.key;
+  return mergedProperty;
+}
+
```

Comment:
what the method does is to wrap base and extras in an object to make sure deepMerge operates as expected when running on simple types